apps/formula1/formula_1.star

```diff
@@ -0,0 +1,280 @@
+"""
+Applet: Formula 1
+Summary: Next F1 Race Location
+Description: Shows Time date and location of Next F1 race.
+Author: AmillionAir
+"""
+
+load("render.star", "render")
+load("http.star", "http")
+load("encoding/json.star", "json")
+load("time.star", "time")
+load("encoding/base64.star", "base64")
+load("schema.star", "schema")
+load("cache.star", "cache")
+
+def main(config):
+    #Time information for date and timezones
+    timezone = config.get("timezone") or "America/Chicago"
+    now = time.now().in_location(timezone)
+    Year = now.format("2006")
+
+    #Add Caching
+    f1_cached = cache.get("f1_rate")
+
+    if f1_cached != None:
+        print("Hit! Displaying cached data.")
+        f1_data = json.decode(f1_cached)
+    else:
+        print("Miss! Calling F1 Track data.")
+
+    #Set API URLS
+    F1_URL = "http://ergast.com/api/f1/" + Year + "/next.json"
+
+    F1_COUNTRY = http.get(F1_URL).json()["MRData"]["RaceTable"]["Races"][0]["Circuit"]["Location"]["country"]
+    F1_LOC = http.get(F1_URL).json()["MRData"]["RaceTable"]["Races"][0]["Circuit"]["Location"]["locality"]
+    F1_DATE = http.get(F1_URL).json()["MRData"]["RaceTable"]["Races"][0]["date"]
+    F1_TIME = http.get(F1_URL).json()["MRData"]["RaceTable"]["Races"][0]["time"]
+    F1_ROUND = http.get(F1_URL).json()["MRData"]["RaceTable"]["Races"][0]["round"]
```

Comment:
Could we assign the results of `http.get(F1_URL).json()` to a variable so we don't have to make 5x requests?

Reply:
DONE!